src/pages/index.js

```diff
@@ -0,0 +1,274 @@
+                            this.onSearchChange(e);
+                        }}
+                    />
+                    <Flex justify="space-between" mt={4}>
+                        <Box>
+                            <Text align="left" color="muted" f={3}>
+                                {filteredClubs.length} club{hasOneResult ? "" : "s"}{" "}
+                                {showAllClubs ? `match${hasOneResult ? "es" : ""} “${searchValue}”` : `found within ${searchRadius} ${useImperialSystem ? "mile" : "kilometer"}${searchRadius === 1 ? "" : "s"} from ${formattedAddress || "you"}`}
+                            </Text>
+                        </Box>
+                        <Settings
+                            searchRadius={parseInt(searchRadius)}
+                            showAllClubs={showAllClubs}
+                            useImperialSystem={useImperialSystem}
+                            style={{ marginRight: theme.space[2] }}
+                            onRadiusChange={e => {
+                                e.persist();
+                                this.setState({ searchRadius: e.target.value });
+                                this.onRadiusChange(e);
+                            }}
+                            onSystemChange={this.onSystemChange}
+                            onViewChange={this.onViewChange}
+                        />
+                    </Flex>
+                    <Flex justify={hasNoResults ? "center" : "initial"} py={4} style={{ marginLeft: -theme.space[2], marginTop: -theme.space[2] }} wrap>
+                        {
+                            filteredClubs.map(club => (
+                                <LazyLoad key={club.id} height={0} offset={100} once overflow>
+                                    <ClubCard
+                                        data={club}
+                                        distance={!showAllClubs && geolib.getDistance({ latitude: searchLat, longitude: searchLng }, club)}
+                                        showAllClubs={showAllClubs}
+                                        useImperialSystem={useImperialSystem}
+                                    />
+                                </LazyLoad>
+                            ))
+                        }
+                        {hasNoResults && <NoClubsFound />}
+                    </Flex>
+                </Container>
+                <Footer />
+            </Fragment>
+        );
+    }
+
+    onRadiusChange(e) {
+        const nextSearchRadius = parseInt(e.target.value);
+        const { clubs, searchLat, searchLng, useImperialSystem } = this.state;
+        this.setState({ searchRadius: nextSearchRadius });
+        this.setParams({ r: nextSearchRadius });
+        if(searchLat && searchLng) {
+            const filteredClubs = this.getFilteredClubs(clubs, {
+                searchLat: searchLat,
+                searchLng: searchLng,
```

Comment:
es6 tip: if the value is the same as the key, remove the colon + value, it will automatically be set